helpers/lib_tasks_pytest.py

```diff
@@ -874,6 +874,121 @@ def run_coverage_report(  # type: ignore
         _publish_html_coverage_report_on_s3(aws_profile)
 
 
+def _get_inclusion_settings(target_dir: str) -> Tuple[str, Optional[str]]:
+    """
+    Determine include/omit glob patterns for the coverage report.
+
+    :param target_dir: directory for coverage stats; use "." to indicate all directories
+    :return: tuple (include_in_report, exclude_from_report) where
+        - include_in_report is the glob pattern to include
+        - exclude_from_report is a comma‑separated glob pattern to omit, or ``None``
+    """
+    if target_dir == ".":
+        include_in_report: str = "*"
+        exclude_from_report: Optional[str] = None
```

Comment:
Let's avoid type hints for variables that are not part of a function or method interface.

Reply:
Same everywhere.